### DISCUSSION TITLE
What would be the most powerful options?

### DISCUSSION THREAD

**Discussion by leuamS** [86Box/86Box#2076] [Q&A] [answered]
What are the most powerful options you can select on a machine?

**Comment by Cacodemon345** [comment]
It depends on your hardware.

**Comment by leuamS** [comment]
I don't mean the most powerful hardware I can run on my PC, I just mean the most powerful options in general

**Comment by Cacodemon345** [comment, marked as answer]
VIA Cyrix III @ 733 MHz
Socket 370 machines
AGP 3dfx Voodoo 3 3000
3GB RAM with certain Socket 370 machines.
Crystal CS4297A (AC97 device, only available on machines where ISA is unavailable).
3.5" 2.88MB floppy drives
PCI Realtek RTL8029AS

**Reply by leuamS** [reply]
cool

**Reply by jadasse** [reply]
I thought the two PCI AMD network cards were more advanced

**Reply by leuamS** [reply]
Falcon Northwest Voodoo 3500 SE is the new most powerful graphics card

**Reply by Flo9696** [reply]
What Socket 370 machines can you get 3GB RAM? 

I only found Socket 370 machines that supports max 2GB RAM.
